tl.tl

```diff
@@ -1837,6 +1853,9 @@ do
       i, lhs = P(ps, i)
       i, lhs = E(ps, i, lhs, 0)
       if lhs then
+         if node_is_require_call(lhs) then
```

Comment:
this test can be moved inside the handler of @funcall, so that it doesn't hit for every expression